website/source/docs/agent/configuration/server.html.md

```diff
@@ -139,20 +139,31 @@ server {
   useful for cases where we know the address will become available eventually.
   Use `retry_join` with an array as a replacement for `start_join`, **do not use
   both options**. See the [server address format](#server-address-format)
-  section for more information on the format of the string.
+  section for more information on the format of the string. This field is
+  deprecated in favor of [server_join](#server_join).
+
+  Note that `retry_join` can be defined for only servers as a command-line
+  flag (clients are only able to define via the client configuration).
+
+   ```sh
+   $ nomad agent -retry-join "127.0.0.1:4648"
```

Comment:
This should go here: https://www.nomadproject.io/docs/commands/agent.html#retry-join